<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWidgetAlbum</class>
 <widget class="QWidget" name="MusicWidgetAlbum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#tabWidget {
	border: 0;
}

#tab, #tab_2 {
	background-color: #ffffff;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="buttonRevert">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Album has changed. Click to revert changes.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../MediaElch.qrc">
         <normaloff>:/img/arrow_circle_left.png</normaloff>:/img/arrow_circle_left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="albumName">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #224f7f;
</string>
       </property>
       <property name="text">
        <string>Album Name</string>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border-radius: 0;
	background-color: transparent;
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Information</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Path</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="path">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="artist">
              <property name="item" stdset="0">
               <string notr="true">artist</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Artist</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="title">
              <property name="item" stdset="0">
               <string notr="true">title</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="label">
              <property name="item" stdset="0">
               <string notr="true">label</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Label</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Rating</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Year</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Release Date</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Review</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QTextEdit" name="review"/>
            </item>
            <item row="7" column="1">
             <widget class="QSpinBox" name="year">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>2050</number>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="releaseDate">
              <property name="item" stdset="0">
               <string notr="true">releaseDate</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QDoubleSpinBox" name="rating">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>MusicBrainz Album ID</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="musicBrainzAlbumId">
              <property name="item" stdset="0">
               <string notr="true">mbAlbumId</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>MusicBrainz Release Group ID</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="musicBrainzReleaseGroupId">
              <property name="item" stdset="0">
               <string notr="true">mbReleaseGroupId</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Extended</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,2,0">
          <item>
           <widget class="TagCloud" name="genreCloud" native="true">
            <property name="item" stdset="0">
             <string notr="true">genre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TagCloud" name="styleCloud" native="true">
            <property name="item" stdset="0">
             <string notr="true">style</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TagCloud" name="moodCloud" native="true">
            <property name="item" stdset="0">
             <string notr="true">mood</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Booklet</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="font">
               <font>
                <family>Helvetica Neue</family>
               </font>
              </property>
              <property name="text">
               <string>Hint: Closed images will be deleted on save.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnAddExtraFanart">
              <property name="text">
               <string>Add Images</string>
              </property>
              <property name="dropShadow" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">color: #ebebeb;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="artWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelCover">
             <property name="text">
              <string>Cover</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ClosableImage" name="cover">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Click to Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../MediaElch.qrc">:/img/placeholders/poster.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="clickable" stdset="0">
              <bool>true</bool>
             </property>
             <property name="myFixedHeight" stdset="0">
              <number>180</number>
             </property>
             <property name="title" stdset="0">
              <string>Cover</string>
             </property>
             <property name="dropShadow" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelDiscArt">
             <property name="text">
              <string>Disc Art</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ClosableImage" name="discArt">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Click to Change</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../MediaElch.qrc">:/img/placeholders/cd_art.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="clickable" stdset="0">
              <bool>true</bool>
             </property>
             <property name="myFixedHeight" stdset="0">
              <number>180</number>
             </property>
             <property name="title" stdset="0">
              <string>Disc Art</string>
             </property>
             <property name="dropShadow" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagCloud</class>
   <extends>QWidget</extends>
   <header>smallWidgets/TagCloud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClosableImage</class>
   <extends>QLabel</extends>
   <header>smallWidgets/ClosableImage.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
